zerver/openapi/zulip.yaml

```
@@ -6200,7 +6200,13 @@ paths:
             subscribed to appropriate channels or use a shared history
             search narrow with this endpoint.
 
-            **Changes**: In Zulip 9.0 (feature level 250), narrows gained support
+            **Changes**: In Zulip 9.0 (feature level 263), narrows gained support for a new
+            filter - `with`, taking a message ID as operand. If the message corresponding to
+            the message ID exists and is visible to the user, then it replaces any "conversation"
+            filters — channel/stream, topic conversation — with filters corresponding to the
+            conversation the message is in. Else, the `with` operator is just ignored.
```

Rather than duplicate the details of these changes here and in the other endpoints that take a narrow, I think it'd be best to just have the changelog in one place and point to there. So for example:
```suggestion
            **Changes**: See changelog at [Construct a narrow](/api/construct-narrow).
```

but as a prep commit replacing the existing changelog here (and at the other endpoints).

That avoids the risk of the details getting out of sync between the different descriptions, and reduces the burden for updating or editing them. Conversely, I think anyone who's getting into the details of constructing a narrow is going to have to follow the link to `/api/construct-narrow` in any case.

---

Preparatory PR - #30244